src/alto/app/estimator.py

```diff
@@ -0,0 +1,76 @@
+"""
+alto-estimator.py
+
+Estimates the throughput of a number of different flows.
+
+Can be used in two ways:
+
+1. From other Python code. Call do_request(input_dict, alto_server)
+    where input_dict is a Python dict representing the "endpoints-flows"
+    argument, and where alto_server is the hostname of the ALTO server.
+2. As a standalone script. In this case, --alto-server gives the hostname
+    of the ALTO server, and --flows gives a path to an input file.
+    The flows file is of the form
+        SRC1 -> DST1 DST2 DST3 ...
+        SRC2 -> DST4 DST5 DST6 ...
+        ...
+
+Note that, due to time constraints, this script does not make use of the
+input compression made possible by Request Design 2.
+"""
+
+import json
+import requests
+
+def input_to_json(input_str):
+    input_lines = input_str.splitlines()
+    # no compression; TODO: compress

```

Comment:
Suggest:
- Start the sentence with "TODO:"
- For every TODO, there should be a Github ticket and in the TODO comment, we should put the URL/link to that ticket